nsxt/resource_nsxt_vpc.go

```diff
@@ -115,6 +115,25 @@ var vpcSchema = map[string]*metadata.ExtendedSchema{
 			OmitIfEmpty:  true,
 		},
 	},
+	"quotas": {
```

Comment:
In this case could it be worth to deviate from API naming and call this attribute "ip_block_quotas"?

Reply:
There could be other types of quotas in the future